src/theme.js

```diff
@@ -1,243 +1,252 @@
+      "list.activeSelectionBackground"  : color.neutral.muted,
+      "list.focusForeground"            : color.fg.default,
+      "list.focusBackground"            : color.accent.subtle,
+      "list.inactiveFocusBackground"    : color.accent.subtle,
+      "list.highlightForeground"        : color.accent.fg,
+
+      "tree.indentGuidesStroke": color.border.muted,
+
+      "notificationCenterHeader.foreground": color.fg.muted,
+      "notificationCenterHeader.background": color.canvas.subtle,
+      "notifications.foreground"           : color.fg.default,
+      "notifications.background"           : color.canvas.overlay,
+      "notifications.border"               : color.border.default,
+      "notificationsErrorIcon.foreground"  : color.danger.fg,
+      "notificationsWarningIcon.foreground": color.attention.fg,
+      "notificationsInfoIcon.foreground"   : color.accent.fg,
+
+      "pickerGroup.border"    : color.border.default,
+      "pickerGroup.foreground": color.fg.muted,
+      "quickInput.background" : color.canvas.overlay,
+      "quickInput.foreground" : color.fg.default,
+
+      "statusBar.foreground"             : color.fg.muted,
+      "statusBar.background"             : color.canvas.default,
+      "statusBar.border"                 : color.border.default,
+      "statusBar.noFolderBackground"     : color.canvas.default,
+      "statusBar.debuggingBackground"    : color.danger.emphasis,
+      "statusBar.debuggingForeground"    : color.fg.onEmphasis,
+      "statusBarItem.prominentBackground": color.canvas.subtle,
+
+      "editorGroupHeader.tabsBackground": color.canvas.inset,
+      "editorGroupHeader.tabsBorder"    : color.border.default,
+      "editorGroup.border"              : color.border.default,
+
+      "tab.activeForeground"        : color.fg.default,
+      "tab.inactiveForeground"      : color.fg.muted,
+      "tab.inactiveBackground"      : color.canvas.inset,
+      "tab.activeBackground"        : color.canvas.default,
+      "tab.hoverBackground"         : color.canvas.default,
+      "tab.unfocusedHoverBackground": color.neutral.subtle,
+      "tab.border"                  : color.border.default,
+      "tab.unfocusedActiveBorderTop": color.border.default,
+      "tab.activeBorder"            : color.canvas.default,
+      "tab.unfocusedActiveBorder"   : color.canvas.default,
+      "tab.activeBorderTop"         : color.primer.border.active,
+
+      "breadcrumb.foreground"               : color.fg.muted,
+      "breadcrumb.focusForeground"          : color.fg.default,
+      "breadcrumb.activeSelectionForeground": color.fg.muted,
+      "breadcrumbPicker.background"         : color.canvas.overlay,
+
+      "editor.foreground"                 : color.fg.default,
+      "editor.background"                 : color.canvas.default,
+      "editorWidget.background"           : color.canvas.overlay,
+      "editor.foldBackground"             : color.neutral.muted,
```

Comment:
Here I'm not sure. I think this color needs to be semi-transparent otherwise it would cover something else. As of today `color.neutral.muted` is semi-transparent and should work, but that might change in the future. So I thought it's safer to add the semi-transparency here in the VS Code theme.

Reply:
I've reverted those to ones with alpha for safety